<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Dialog</class>
  <widget class="QDialog" name="Dialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>242</width>
        <height>362</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>242</width>
        <height>362</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>242</width>
        <height>362</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Calculator</string>
    </property>
    <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);
        background-color: rgb(0, 0, 0);
        color: rgb(0, 0, 0);
      </string>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
        <number>0</number>
      </property>
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>
      <item>
        <widget class="HFrame" name="frameMain">
          <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="leftMargin">
              <number>0</number>
            </property>
            <property name="topMargin">
              <number>0</number>
            </property>
            <property name="rightMargin">
              <number>0</number>
            </property>
            <property name="bottomMargin">
              <number>0</number>
            </property>
            <item>
              <widget class="QLCDNumber" name="lcdDisplay">
                <property name="minimumSize">
                  <size>
                    <width>180</width>
                    <height>60</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>16777215</width>
                    <height>60</height>
                  </size>
                </property>
                <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);
                    color: rgb(186, 189, 182);
                    font: 75 16pt &quot;Sawasdee&quot;;
                  </string>
                </property>
                <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                  <number>0</number>
                </property>
                <property name="digitCount">
                  <number>8</number>
                </property>
                <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QFrame" name="frame">
                <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                </property>
                <property name="frameShape">
                  <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                  <number>0</number>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                  <property name="leftMargin">
                    <number>0</number>
                  </property>
                  <property name="topMargin">
                    <number>0</number>
                  </property>
                  <property name="rightMargin">
                    <number>0</number>
                  </property>
                  <property name="bottomMargin">
                    <number>0</number>
                  </property>
                  <property name="spacing">
                    <number>0</number>
                  </property>
                  <item row="0" column="4">
                    <widget class="QToolButton" name="btnDivision">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">background-color: rgb(253, 128, 8);
                          color: rgb(255, 255, 255);
                          font: 25pt &quot;Arial&quot;;
                          font-weight: bold;
                        </string>
                      </property>
                      <property name="text">
                        <string>รท</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="2" column="0">
                    <widget class="QToolButton" name="btn4">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>4</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="1" column="0">
                    <widget class="QToolButton" name="btn7">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>7</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="1" column="4">
                    <widget class="QToolButton" name="btnMultiplication">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">background-color: rgb(253, 128, 8);
                          color: rgb(255, 255, 255);
                          font: 20pt &quot;Arial&quot;;
                          font-weight: bold;
                        </string>
                      </property>
                      <property name="text">
                        <string>x</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="1" column="1">
                    <widget class="QToolButton" name="btn8">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>8</string>
                      </property>
                    </widget>
                  </item>
                  <item row="3" column="0">
                    <widget class="QToolButton" name="btn1">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>1</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="3">
                    <widget class="QToolButton" name="btnPercent">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(46, 52, 54);
                        </string>
                      </property>
                      <property name="text">
                        <string>%</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="3" column="3">
                    <widget class="QToolButton" name="btn3">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>3</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="2" column="4">
                    <widget class="QToolButton" name="btnMinus">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">background-color: rgb(253, 128, 8);
                          color: rgb(255, 255, 255);
                          font: 30pt &quot;Arial&quot;;
                          font-weight: bold;
                        </string>
                      </property>
                      <property name="text">
                        <string>-</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="3" column="1">
                    <widget class="QToolButton" name="btn2">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>2</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="0">
                    <widget class="QToolButton" name="btnAC">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(46, 52, 54);
                        </string>
                      </property>
                      <property name="text">
                        <string>AC</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="1" column="3">
                    <widget class="QToolButton" name="btn9">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>9</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="4" column="4">
                    <widget class="QToolButton" name="btnEqual">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">background-color: rgb(253, 128, 8);
                          color: rgb(255, 255, 255);
                          font: 20pt &quot;Arial&quot;;
                          font-weight: bold;
                        </string>
                      </property>
                      <property name="text">
                        <string>=</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="2" column="1">
                    <widget class="QToolButton" name="btn5">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>5</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="4" column="0" colspan="2">
                    <widget class="QToolButton" name="btn0">
                      <property name="minimumSize">
                        <size>
                          <width>120</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>0</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="2" column="3">
                    <widget class="QToolButton" name="btn6">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string>6</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="4" column="3">
                    <widget class="QToolButton" name="btnDecimal">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 20pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(85, 87, 83);
                        </string>
                      </property>
                      <property name="text">
                        <string/>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="3" column="4">
                    <widget class="QToolButton" name="btnPlus">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">background-color: rgb(253, 128, 8);
                          color: rgb(255, 255, 255);
                          font: 20pt &quot;Arial&quot;;
                          font-weight: bold;
                        </string>
                      </property>
                      <property name="text">
                        <string>+</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="1">
                    <widget class="QToolButton" name="btnSignal">
                      <property name="minimumSize">
                        <size>
                          <width>60</width>
                          <height>60</height>
                        </size>
                      </property>
                      <property name="styleSheet">
                        <string notr="true">font: 75 14pt &quot;Arial&quot;;
                          color: rgb(238, 238, 236);
                          background-color: rgb(46, 52, 54);
                        </string>
                      </property>
                      <property name="text">
                        <string>+/-</string>
                      </property>
                      <property name="toolButtonStyle">
                        <enum>Qt::ToolButtonTextOnly</enum>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>HFrame</class>
      <extends>QFrame</extends>
      <header>hspylib.modules.qt.promotions.hframe</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>btnAC</tabstop>
    <tabstop>btnSignal</tabstop>
    <tabstop>btnPercent</tabstop>
    <tabstop>btnDivision</tabstop>
    <tabstop>btn7</tabstop>
    <tabstop>btn8</tabstop>
    <tabstop>btn9</tabstop>
    <tabstop>btnMultiplication</tabstop>
    <tabstop>btn4</tabstop>
    <tabstop>btn5</tabstop>
    <tabstop>btn6</tabstop>
    <tabstop>btnMinus</tabstop>
    <tabstop>btn1</tabstop>
    <tabstop>btn2</tabstop>
    <tabstop>btn3</tabstop>
    <tabstop>btnPlus</tabstop>
    <tabstop>btn0</tabstop>
    <tabstop>btnDecimal</tabstop>
    <tabstop>btnEqual</tabstop>
  </tabstops>
  <resources/>
  <connections/>
</ui>
